common/theme.c

```diff
@@ -123,3 +135,64 @@ void themeStartup(ThemePreset preset) {
     }
     config_destroy(&cfg);
 }
+
+void GetThemePathFromConfig(char* themePath) {
```

Comment:
Max size of themePath should be specified here, which is then used with the strncpy.

Reply:
how so?

Reply:
char themePath[PATH_MAX] ?

Reply:
(char* themePath, size_t size)

Reply:
done